src/sdk/Audio/SpeakerAudioDestination.ts

```diff
@@ -52,6 +52,7 @@ export class SpeakerAudioDestination implements IAudioDestination, IPlayer {
         this.privId = audioDestinationId ? audioDestinationId : createNoDashGuid();
         this.privIsPaused = false;
         this.privIsClosed = false;
+        this.privAudio = new Audio();
```

Comment:
The `undefined` value of  `privaAudio` means the playback is not supported (e.g. unsupported format, see line 134). This would break some current logic. So pls revert.

Reply:
@yulin-li can you please elaborate your response? From what I can see calling method `format` will re-initialize `privAudio` so it doesn't matter if I also set it constructor. I'm only trying to understand if there's a better approach than re-adding `undefined` checks (following null object pattern). Thanks.